psw/urts/loader.cpp

```diff
@@ -919,12 +919,18 @@ int CLoader::load_enclave_ex(SGXLaunchToken *lc, bool debug, const metadata_t *m
         case SGX_ERROR_ENCLAVE_LOST:     //caused by loading enclave while power transition occurs
             break;
 
-            //If memroy map conflict occurs, we only retry 3 times.
+            //If memroy map fail or conflict occurs, we only retry 3 times.
+        case SGX_ERROR_MEMORY_MAP_FAILURE:
```

Comment:
Can we merge these two `case`s to simplify the code?
```
case SGX_ERROR_MEMORY_MAP_FAILURE:
case SGX_ERROR_MEMORY_MAP_CONFLICT:
     ......
     break;
```

Reply:
done